docs/pages/includes/insecure-certificate.mdx

```diff
@@ -1,8 +1,8 @@
 <Admonition type="tip" title="Insecure Certificate Error">
   If you encounter an "Insecure Certificate Error" (or equivalent warning) that prevents the Teleport Web UI from opening, you can:
 
-  1. Open the URL in Safari.
-  2. Use the Chrome flag `--ignore-certificate-errors` instead.
+  1. Open the URL in Safari or Firefox.
```

Comment:
Safari and Firefox must display similar warnings, right?

Also, for Chrome, rather than changing `chrome://flags` what about mentioning the [`thisisunsafe`](https://miguelpiedrafita.com/chrome-thisisunsafe) trick?